src/components/ScholarshipAmountField.test.jsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
+import UserEvent from '@testing-library/user-event';
 import ScholarshipAmountField from './ScholarshipAmountField';
 
-test('renders radio buttons', () => {
-  const labels = ['Full Tuition', /Fixed Amount:.*/, /Range:.*/, 'Unknown'];
-  render(
-    <ScholarshipAmountField onTypeChange={() => {}} updateAmount={() => {}} />
+function renderWithAmountType(type) {
+  return render(
+    <ScholarshipAmountField
+      onTypeChange={() => {}}
+      updateAmount={() => {}}
+      amountType={type}
+    />
   );
+}
 
-  expect(screen.getAllByRole('radio')).toHaveLength(4);
-  labels.forEach((name) =>
-    expect(screen.getByRole('radio', { name })).toBeInTheDocument()
-  );
+test('renders select options', () => {
+  renderWithAmountType('FIXED');
```

Comment:
Use the [AmountType](https://github.com/beyondhb1079/s4us/blob/main/src/types/AmountType.ts) constant here and elsewhere.